obstacle_spawner:
  ros__parameters:
    inflation_radius: 1.0   # meters
    carla_host: "127.0.0.1"
    carla_port: 2000
    publish_rate_hz: 10.0
    vehicles:
      car1:
        blueprint: "vehicle.nissan.patrol"
        x: 26.63530158996582
        y: 141.5
        z: 0.1
        yaw: 0.0
      car2:
        blueprint: "vehicle.sprinter.mercedes"
        x: 109.19975280761719
        y: 51.77189254760742
        z: 0.1
        yaw: -90.0

      car3:
        blueprint: "vehicle.sprinter.mercedes"
        x: 39.86615753173828
        y: -68.25939178466797
        z: 0.1
        yaw: 180
